todoApp/src/Components/TodoCreate/TodoCreate.tsx

```diff
@@ -0,0 +1,43 @@
+import "./TodoCreate.css";
+import TodoList from "../TodoList/TodoList.tsx";
+
+// Define the props for TodoCreate
+interface Todo {
+    id: string;
+    todo: string;
+    category: string;
+    done: boolean;
+    editable: boolean;
+}
+
+interface TodoCreateProps {
```

Comment:
create separate type.ts file

```suggestion
TodoCreate.types.ts
```